analyzer-comments/elm/feedback_request.md

```diff
@@ -0,0 +1,4 @@
+# feedback request
+
+If this automated feedback doesn't look right, please [open an issue in the `exercism/elm-analyzer` repository](https://github.com/exercism/elm-analyzer/issues?q=is%%3Aissue+is%%3Aopen+sort%%3Aupdated-desc).
```

Comment:
Are you sure that link is correct?

Also, it might be worthwhile to make a reference link instead:  Updating it later will mean a cleaner diff, less noise.

Reply:
The link is correct, because the % are being escaped. It's really confusing, see #2196.
Thank you for the reference link suggestion.

Reply:
There you go.